lib/opentelemetry/trace/exporter/otlp.lua

```diff
@@ -64,11 +91,15 @@ function _M.export_spans(self, spans)
             status = span.status,
         })
     end
-    self.client:do_request(pb.encode(body))
+    self:call_collector(pb.encode(body))
 end
 
 function _M.shutdown(self)
 
 end
 
-return _M
\ No newline at end of file
+if _TEST then
```

Comment:
This was [mentioned as a way to test private functions in the busted repository](https://github.com/Olivine-Labs/busted/issues/605#issuecomment-511490382). I'm open to other ideas!

Reply:
Exactly! Your question made me realize there was a bug, which I addressed here 18a33a1fe9c32e6afc2834d49077c5c614963df6

Reply:
Suggest call `export_spans(mock_spans)` instead of `cb:call_collector("ok")` in spec/trace/exporter/otlp_spec.lua.
So we can remove `_TEST`.